script.js

```diff
@@ -0,0 +1,57 @@
+const bookTitle = document.querySelector('#bookName');
+const bookAuthor = document.querySelector('#bookAuthor');
+const addBk = document.querySelector('.add-btn');
+const booksWrapper = document.querySelector('.books-list');
+
+let booksList = JSON.parse(localStorage.getItem('bookStorage')) || [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = title;
+  }
+}
+
+function showBook(title, author) {
+  booksWrapper.innerHTML += `
+  <div>
+    <p>${title}</p>  
+    <p>${author}</p>
+    <button type='button' class="remove-btn" id="${title}">Remove</button>
+  </div>`;
+  const removeBtn = document.querySelectorAll('.remove-btn');
+  removeBtn.forEach((button) => {
+    button.addEventListener('click', () => {
+      booksList = booksList.filter((book) => book.id !== button.id);
```

Comment:
- [ ] Good job so far. For removing a book you're filtering by title, which is great but if two books have the same title it will remove both of them by clicking a remove button, I suggest you make the id more unique.